tests/ops/test_tensor_utils.py

```diff
@@ -22,3 +22,18 @@ def test_block_diag(batch_size, block_size):
         bottom, top = k * block_size[0], (k + 1) * block_size[0]
         left, right = k * block_size[1], (k + 1) * block_size[1]
         assert_equal(b[bottom:top, left:right], m[k])
+
+
+@pytest.mark.parametrize('T', [2, 3, 4, 5, 10])
+@pytest.mark.parametrize('batch_shape', [(), (4,), (2, 3)], ids=str)
+def test_conv1d_fft(batch_shape, T):
+    signal = torch.randn(T)  # not batched
+    kernel = torch.randn(*batch_shape, T).exp()  # batched
+    actual = conv1d_fft(signal, kernel)
+
+    # FIXME is this correct?
```

Comment:
I think it is better to compare against https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.fftconvolve.html#scipy.signal.fftconvolve.

Reply:
The scipy.signal.convolve should agree with scipy.signal.fftconvolve.

Reply:
Actually, np.convolve seems like an even better gold standard.